libclc/ptx-nvidiacl/libspirv/workitem/get_sub_group_size.cl

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <spirv/spirv.h>
+_CLC_DEF _CLC_OVERLOAD uint __spirv_SubgroupId();
+_CLC_DEF _CLC_OVERLOAD uint __spirv_NumSubgroups();
+_CLC_DEF _CLC_OVERLOAD uint __spirv_SubgroupMaxSize();
```

Comment:
Probably, we should move these forward declarations to corresponding header file similar to https://github.com/intel/llvm/tree/sycl/libclc/generic/include/spirv/workitem to following existing style.

Reply:
Makes sense.  I assumed this was an auto-generated SPIR-V header coming in from somewhere else.  Fixed in fcc055a9cfe1c8b935eb6d1c6e7a221f4893edff.